### PR TITLE
lint fix

### PR DESCRIPTION
**Pull request by rakeshgm**
metrodr-script.sh:
    - fixing script as per shellcheck for linting
    - format indentation 
  
  dev-rook-cluster-external.yaml:
    - document-start section added
 

Signed-off-by: rakeshgm <rakeshgm@redhat.com>

### CODE REVIEW

File: metrodr-script.sh [hunk 1]
```diff
@@ -12,43 +12,46 @@ scriptdir="$basedir/ramen-1"
...
-    export NAMESPACE
-  else
-    kubectl --context "$1" create ns "$2"
-    NAMESPACE="$2"
-    export NAMESPACE
-  fi
+        if kubectl --context "$1" get ns -o name | grep -q "$2"
+        then
+                echo "namespace exists, continuing"
+        NAMESPACE="$2"
```
Comment (raghavendra-talur) [inline metrodr-script.sh]: The indentation here seems off. This and the next line should be at the same indentation level as the echo line above.

File: metrodr-script.sh [hunk 2]
```diff
@@ -12,43 +12,46 @@ scriptdir="$basedir/ramen-1"
...
+        export NAMESPACE
+        else
+        kubectl --context "$1" create ns "$2"
```
Comment (raghavendra-talur) [inline metrodr-script.sh]: Add indentation here.

**Review by raghavendra-talur — CHANGES_REQUESTED** [review]
The changes look great. @raghavendrabhat and I reviewed the changes together. We have one small comment to be addressed.

Also, please split the PR into two commits, one for the script changes and another for the yaml file.

File: metrodr-script.sh (hunk 2 above)
Comment (rakeshgm) [reply metrodr-script.sh]: indentation added

File: metrodr-script.sh (hunk 1 above)
Comment (rakeshgm) [reply metrodr-script.sh]: corrected indentation level

**Comment by rakeshgm** [comment]
I thought of adding another commit to this PR to move `metrodr-script.sh` to `hack` dir, but then again this PR is only for lint fixing. I will send another PR once this is merged. let me know your thoughts on this.

**Review by raghavendra-talur — APPROVED** [review]

**Pull request merged**